build/configs/rtl8721csm/Make.defs

```diff
@@ -258,6 +259,11 @@ else
 endif
 endif
 
+ifeq ($(CONFIG_LIBCXX_EXCEPTION),y)
+#this symbol should be present in all the cpp binaries
+LDELFFLAGS += --require-defined __gnu_Unwind_Find_exidx --force-group-allocation
```

Comment:
Let us add more comments about why we need these two flags. It will be useful for reference in future.